tests/Rule/LatteTemplatesRule/EngineBootstrap/LatteTemplatesRuleForEngineBootstrapTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Efabrica\PHPStanLatte\Tests\Rule\LatteTemplatesRule\PresenterWithoutModule;
+
+use Efabrica\PHPStanLatte\Tests\Rule\LatteTemplatesRule\LatteTemplatesRuleTest;
+
+final class LatteTemplatesRuleForEngineBootstrapTest extends LatteTemplatesRuleTest
+{
+    public static function getAdditionalConfigFiles(): array
+    {
+        return [
+            __DIR__ . '/../../../../rules.neon',
+            __DIR__ . '/../../../config.neon',
+            __DIR__ . '/config.neon',
+        ];
+    }
+
+    public function testFilters(): void
+    {
+        $this->analyse([__DIR__ . '/Fixtures/FiltersPresenter.php'], [
+            [
+                'Trying to invoke mixed but it\'s not a callable.',
```

Comment:
do we need this second error? the second one is nice and I like it :)

Reply:
First error is side effect. We cannot ignore it because exactly same error could be triggered by user code. But maybe we can generate different code that will not trigger first error when filter is unknown.

Reply:
Yeah, we wrote about it before. I recall... So if we can't ignore it, then we have to leave with it :(

Reply:
I will try to solve it in the future